csi/deployment.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	DefaultCSIAttacherImage            = "longhornio/csi-attacher:v2.0.0"
-	DefaultCSIProvisionerImage         = "longhornio/csi-provisioner:v1.4.0"
+	DefaultCSIProvisionerImage         = "longhornio/csi-provisioner:v1.6.0"
```

Comment:
Will this be upgraded automatically if the user upgrades Longhorn?

Also, does provisioner v1.6.0 has requirements on Kubernetes version?

Reply:
Same kubernetes [minimum version 1.13](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.6.0) requirement as  the currently used [provisioner](https://github.com/kubernetes-csi/external-provisioner/releases/tag/v1.4.0)

For the default installation yes, but it's possible that the users have pinned their images (airgap),
in which case they need to update their images anyway if they want to use the csi-snapshot support.